src/store/reducers/shardsWorkload/shardsWorkload.ts

```diff
@@ -63,27 +65,28 @@ function createShardQueryHistorical(
     PeakTime,
     InFlightTxCount,
     IntervalEnd
-FROM \`.sys/top_partitions_one_hour\`
+FROM \`${database}/.sys/top_partitions_one_hour\`
```

Comment:
If database is empty (as far as I can see - it can be '' because we are checking its value in ternary above) - will /.sys/... be okay as select source?

Reply:
`.sys` will work, `/.sys` - won't. However, we alway sent request to specific database, otherwise queries may not work at all (currently I have errors when I sent requests without defined database).

I think it's better to remove ternaries - they have no sense.

Also I found more complex issue, we don't determine db correctly for nested entities, so for nested databases we have wrong db anyway. So actually, there is no difference, whether we pass database or not, `.sys` path could be relative when db is passed. Probably I revert the changes where I add database to `.sys` path

Reply:
Removed unneeded ternaries, database should never be empty or request won't be successful anyway. BTW, these ternaries were not needed even in case of empty string database, YQL processed it correctly.

Used relative paths everywhere: both in query text and api request we use the same param, so it never differs. If we use relative path, the query will use path from the database, that we passed.